terms/live-edit-and-locked-edits.md

```diff
@@ -2,6 +2,8 @@
 
 > “Data that provides information about other data” \[[wikipedia](https://en.wikipedia.org/wiki/Metadata)]
 
+Metadata is defined as data about data. It gives context to the data it describes. Although this is a complete definition of what metadata is, it is not specific enough for most people. So, it can be useful to refine the idea of metadata to more specific subsets: [schemas][schema.md], [vocabularies][vocabulary.md], and ontologies[ontology.md].
+
```

Comment:
We already have the first bit from the wikipedia entry, it's almost an identical repeat. Also, whilst a schema, vocabularies and ontologies are related to, and part of, metadata they definitely don't constitute